network/load-balancer/reference-content/configuring-load-balancer.mdx

```diff
@@ -0,0 +1,159 @@
+
+**HTTPS**: The same method and fields as for an HTTP health check apply, but data transmission is encrypted. If you chose HTTP as the main protocol for the Load Balancer's backend and activated TLS, the Verify Certificate setting you chose there is inherited for health checks. If you chose a different protocol for the Load Balancer's backend, the backend server's certificate will **not** be verified during health checks. See the [dedicated Verify Certificate section](#choosing-a-verify-certificate-setting) for more information.
+
+**MYSQL**: For MYSQL health checks, you are prompted to select the database username. The MYSQL health check sends two MySQL packets to the server: first a Client Authentication packet and then a QUIT packet to properly close the MySQL session. The received MySQL Handshake Initialization packet (and/or Error packet) is then parsed. This basic but useful test does not produce errors or aborted connections on the server. However, bear in mind that it does not check database presence or database consistency (an external check e.g. with xinetd would be required for this).
+    <Message type="tip">
+      The user specified for the health check must be unlocked and authorized without a password. To create a basic limited user in MySQL with optional resource limits:
+        ```
+        CREATE USER '<username>'@'<ip_of_haproxy|network_of_haproxy/netmask>'
+        /*!50701 WITH MAX_QUERIES_PER_HOUR 1 MAX_UPDATES_PER_HOUR 0 */
+        /*M!100201 MAX_STATEMENT_TIME 0.0001 */;
+        ```
+    </Message>
+    <Message type="important">
+    This health check method requires MySQL >=3.22. For older versions, we recommend using a TCP health check.
+    </Message>
+
+**PGSQL**: For PGSQL health checks, you are prompted to select the database username. The check sends a PostgreSQL StartupMessage and waits for either an `Authentication request` or `ErrorResponse` message. This basic but useful test does not produce errors or aborted connections on the server. This check is identical to the `mysql-check`.
+    <Message type="important">
+      There is a known bug with PGSQL health checks where the PGSQL server verison is higher than **14**. A fix is currently underway. In the meantime,  we recommend using a TCP health check for PGSQL servers of version 14 or higher.
+    </Message>
+
+**LDAP**: No additional settings are required for LDAP health checks. LDAP health checks test connection to the LDAP server by sending an a LDAPv3 anonymous simple bind message. This checks the LDAP service is actually up, and not just that a TCP session can be established.
+
+**Redis**: No additional settings are required for REDIS health checks. A PING redis command is sent to the server, and the response is analyzed for the +PONG response message. This checks that the server is correctly communicating with the REDIS protocol, rather than just accepting the TCP connection.
```

Comment:
```suggestion
**Redis**: No additional settings are required for REDIS health checks. A PING redis command is sent to the server, and the response is analyzed for the `+PONG` response message. This checks that the server is correctly communicating with the REDIS protocol, rather than just accepting the TCP connection.
```